packages/pigeon/platform_tests/flutter_null_safe_unit_tests/lib/multiple_arity.gen.dart

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 //
-// Autogenerated from Pigeon (v4.0.2), do not edit directly.
+// Autogenerated from Pigeon (v4.2.7), do not edit directly.
```

Comment:
I figured since I was changing a file in this directory I'd go ahead and check in all the pending updates to the other files here.